### ISSUE TITLE
Gradle 5.0 deprecated api

### ISSUE THREAD

**Issue by jhawk28** [google/protobuf-gradle-plugin#227]
Gradle 4.7 complains about the protobuf plugin using a deprecated API.

```
> Task :<myproject>:generateProto UP-TO-DATE
Using TaskInputs.file() with something that doesn't resolve to a File object has been deprecated and is scheduled to be removed in Gradle 5.0. Use TaskInputs.files() instead.
```

**Comment by pgelinas** [comment]
Should be fixed by #172 and version 0.8.4; latest is 0.8.5, you should update.

**Comment by jhawk28** [comment]
confirmed on 0.8.5
